bokehjs/test/models/tickers/log_ticker.coffee

```diff
@@ -0,0 +1,50 @@
+{expect} = require "chai"
+utils = require "../../utils"
+
+{LogTicker} = utils.require("models/tickers/log_ticker")
+
+describe "LogTicker Model", ->
+
+  describe "LogTicker get_ticks_no_defaults method", ->
+
+    # short range (<2 base orders)
+
+    it "should have three major ticks and zero minor ticks for long range", ->
+      ticker = new LogTicker({desired_num_ticks: 3, num_minor_ticks: 0})
+      ticks = ticker.get_ticks_no_defaults(1, 999, null, 3)
+      expect(ticks.major).to.be.deep.equal([ 1, 10, 100 ])
+      expect(ticks.minor).to.be.deep.equal([ ])
+
+    it "should have three major ticks and five minor ticks for long range", ->
+      ticker = new LogTicker({desired_num_ticks: 3, num_minor_ticks: 1})
+      ticks = ticker.get_ticks_no_defaults(1, 999, null, 3)
+      expect(ticks.major).to.be.deep.equal([ 1, 10, 100 ])
+      expect(ticks.minor).to.be.deep.equal([ 0.1, 1, 10, 100, 1000 ])
+
+    it "should do stuff on short scale", ->
+      ticker = new LogTicker({desired_num_ticks: 3, num_minor_ticks: 2})
+      ticks = ticker.get_ticks_no_defaults(1, 999, null, 3)
+      expect(ticks.major).to.be.deep.equal([ 1, 10, 100 ])
+      # 0.2 is weird, should be 0.5 i think
```

Comment:
We should figure this out but doesn't have to happen here. Just make a separate issue that references this one.